deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -1299,6 +1299,8 @@ spec:
                     Manufacturer: KubeVirt
                     Product: None
                 - name: MACHINETYPE
+                - name: HCO_KV_IO_VERSION
```

Comment:
Is this related?

Reply:
No, we added it in #528 and we forgot to regenerate the CSV there